combat-trainer.lic

```diff
@@ -369,7 +373,29 @@ class LootProcess
       echo 'Detected harvest messaging' if $debug_mode_ct
       @last_ritual = ritual
       waitrt?
-      bput('drop material', 'you discard it')
+	  if !@necro_store
+	    echo "Thanatology:Store: = false, dropping harvested material" if $debug_mode_ct
+        bput('drop material', 'you discard it')
+	  else
+	    echo "Thanatology:Store: = true, checking count of stored material" if $debug_mode_ct
+	    than_count = @thanatology['harvest_count']
+	    than_container = @thanatology['harvest_container']
+	    materials = fput("rummage /C material #{than_container}")
```

Comment:
I recommend you use the `rummage` helper in `common` (something like `rummage('C material', than_container)`).